spec/hbw/features/bp_form/dynamic_conditions_by_datetime_spec.rb

```diff
@@ -0,0 +1,118 @@
+feature 'Control fields with dynamic conditions by datetime', js: true do
+  let(:neutral_option_cell) { find(:xpath, "//*[text() = 'Neutral option']") }
+  let(:disable_fields_cell) { find(:xpath, "//*[text() = 'Disable fields']") }
+  let(:hide_fields_cell)    { find(:xpath, "//*[text() = 'Hide fields']") }
```

Comment:
Use `text:` or `exact_text:` option.

Reply:
This lazy variables are redundant (because of copy-past). I will saw off them.